lib/rspec/sidekiq/matchers/have_enqueued_sidekiq_job.rb

```diff
@@ -1,11 +1,11 @@
 module RSpec
   module Sidekiq
     module Matchers
-      def have_enqueued_job(*expected_arguments)
-        HaveEnqueuedJob.new expected_arguments
+      def have_enqueued_sidekiq_job(*expected_arguments)
+        HaveEnqueuedSidekiqJob.new expected_arguments
       end
```

Comment:
@JonRowe the downsides would not be the same, as if you have the alias you will not be sure which version of the matcher someone is using. Maybe someone will report an issue and then it will turn out the rspec-rails matcher was being called. Also possibly someone might want to use both matchers in different specs.
Perhaps a compromise would be to have an "alias" method with a deprecation warning. And remove it at some point. But taking into account how easy it is to update your specs to reflect the change, I would just rename it and be done with it.